test/powershell/engine/Module/TestModuleManifest.Tests.ps1

```diff
@@ -81,8 +81,7 @@ Describe "Test-ModuleManifest tests" -tags "CI" {
     }
 
     It "module manifest containing valid processed empty rootmodule file type fails: <rootModuleValue>" -TestCases (
-        @{rootModuleValue = "foo.cdxml"; error = "System.Xml.XmlException"},  # fails when cmdlet tries to read it as XML
-        @{rootModuleValue = "foo.xaml"; error = "Modules_WorkflowModuleNotSupported"}   # not supported on PowerShell 6+
```

Comment:
Is there some other test case for invalid rootmodule files that xaml can be added to?

Reply:
Sorry, I don't understand your request. Do you want to add some new tests? If we have gaps in the test we could address this later.